src/analysis/classic/hunter/marksman/normalizers/KillCommandNormalizer.ts

```diff
@@ -0,0 +1,95 @@
+import HIT_TYPES from 'game/HIT_TYPES';
```

Comment:
also probably want to move this to `shared`

Reply:
Done